src/ol/interaction/Interaction.js

```diff
@@ -135,6 +135,7 @@ export function zoomByDelta(view, delta, opt_anchor, opt_duration) {
   const newZoom = view.getConstrainedZoom(currentZoom + delta);
   const newResolution = view.getResolutionForZoom(newZoom);
 
+  view.resolveConstraints(0);
```

Comment:
Is this needed? Or will it still be needed if `ol/View#animate()` handles it (like I suggested in the previous comment)?

Reply:
See other comment